### ISSUE TITLE
Two small newline issues in doc strings in t.a.s.ServiceMaker

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5160]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#5160|
|Type|defect|
|Created|2011-06-26 10:04:14Z|

Another pathetic ticket by typo man. ;)

In one case, a space was missing in a multi line string (leading to a "theconfiguration" and another forgot to mask newline away so the summary misses the important part.

Patch is attached.

Attachments:

* [newlines-fixes.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f7e/f7e82f2217afedce2363d713e18dddb05ee3bb28/62b9b6f44f71b39de9e459189aef1e164ace360e.diff) (743 bytes) - added by hynek on 2011-06-26 10:04:46Z - Patch fixing the mentioned issues. No code changes. 
* [newlines-fixes-wo-backslash.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f7e/f7e82f2217afedce2363d713e18dddb05ee3bb28/66b56541f47575e026153ac64e23712959e3952c.diff) (489 bytes) - added by hynek on 2011-06-26 13:03:31Z - Only the first part of the patch

<details><summary>Searchable metadata</summary>

```
trac-id__5160 5160
type__defect defect
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__typo typo
time__1309082654000000 1309082654000000
changetime__1309093719000000 1309093719000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @hynek|
|-|-|

The first part of this patch is fine.  The second is not.  Making the first line a reasonable summary is not part of the Twisted coding standard.  It's generally too difficult to fit a reasonable summary into 75 characters or fewer.

Also don't forget to add "review" to your tickets when you think they're ready.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_2"></a>|@hynek commented|
|-|-|

Fixed the patch as requested, will open a pydoctor isuee later as discussed on IRC.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks.  Looks good, applying now.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [32184]) Apply patch fixing missing space in application API documentation

Author: hynek
Reviewer: exarkun
Fixes: [#5160](https://github.com/twisted/twisted/issues/5160)

Insert a missing space in the `IServiceMaker.options` API documentation.
